CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+# v2.0.0
+
+Reaction v2.0.0—an API-first, real-time commerce engine built using Node.js, React, and GraphQL. It’s the second major release of our open source commerce software.
+
+This release is coordinated with [Reaction Platform](https://github.com/reactioncommerce/reaction-platform) and is designed to work with the [Reaction](https://github.com/reactioncommerce/reaction) and `[reaction-hydra](https://github.com/reactioncommerce/reaction-hydra)`.
```

Comment:
hydra link is not rendering correctly. See preview here: https://github.com/reactioncommerce/example-storefront/blob/4080d68f545050faa3633f613a24a335fdeade0b/CHANGELOG.md